scripts/qa_crowbarsetup.sh

```diff
@@ -799,8 +799,10 @@ BROADCAST='$net.255'
 EOF
     ifdown br0
     rm -f /etc/sysconfig/network/ifcfg-br0
-    grep -q "^default" /etc/sysconfig/network/routes || \
```

Comment:
> It seems useless (unless there are other routes).

Doesn't this depend on whether the base SLES image for the admin node has a default route in it?  If it doesn't, the `grep` is useless, but should we really be assuming that?  The base image could easily change in the future.  In which case I disagree with dropping the `grep`; [it's better to program defensively](http://en.wikipedia.org/wiki/Robustness_principle).